mender-convert-modify

```diff
@@ -143,7 +143,9 @@ initrd_imagetype=initrd
 EOF
 
   run_and_log_cmd "ln -s ${kernel_imagetype} work/rootfs/boot/kernel"
-  run_and_log_cmd "ln -s ${initrd_imagetype} work/rootfs/boot/initrd"
+  if [ "${initrd_imagetype}" != "" ]; then
+    run_and_log_cmd "ln -s ${initrd_imagetype} work/rootfs/boot/initrd"
+  fi
```

Comment:
@mirzak This I had to add once I upgraded the Ubuntu image. The only changes in the image is adding packages `openssh-server` and `dhcpcd5`, so I don't know how could that have changed the kernel to not use an initrd now.

Do you think this is an incorrect fix?

(the qemu image boots, the tests actually pass)

Reply:
I moved the related commits to #243. Let's discuss there how should this be fixed.